docs/getting-started/quick-start.md

```diff
@@ -2,14 +2,254 @@
 sidebar_position: 2
 ---
 
-# Quick Start
+# Quick start
 
-In this section we are providing a sample application that one can deploy and just use to play around.
+In this quick start, you will see a sample application that you can deploy and
+use to play around. This will allow you to understand how Service Binding
+Operator can be used to simplify the connection between a service, like a
+database and the application.
 
-## K8S
+For this sample, we will be using a PostgreSQL database and a simple application
+which will be the [Spring Boot REST API server][petclinic] sample.
 
-_Here we provided a small sample on K8s_
+To illustrate what we are going to do, here is a visual representation of the
+application we are going to setup.
 
-## OpenShift
+![postgresql-spring-boot](/img/docs/postgresql-spring-boot.png)
 
-_Here we provided a small sample on OpenShift_
\ No newline at end of file
+In this configuration, we will leverage the service binding operator, to collect
```

Comment:
Suggestion: This information should be going under the overview section related to the process part.
Avoid future tense. Make it direct. Something like, "The objective of this configuration is to leverage the service binding Operator to collect ..."

Reply:
Please the current text.. 9d090d950e79e610a67ce1fdaedbb7d3dd825d5a